snare/cloner.py

```diff
@@ -140,6 +140,7 @@ def _make_filename(self, url):
             if data is not None:
                 self.meta[file_name]['hash'] = hash_name
                 self.meta[file_name]['content_type'] = content_type
+                print('Cloned url: ',current_url)
```

Comment:
I'd prefer to keep stdout as clean as possible because if you have thousands of urls it will be super messy, I suggest you think about printing some counts or put some animation